runHarness/Parameters.pm

```diff
@@ -940,7 +940,7 @@ $parameters{"description"} = {
 
 $parameters{"rampUp"} = {
 	"type"    => "=i",
-	"default" => "300",
+	"default" => "420",
```

Comment:
This is becoming a long time.  Is this ramp needed for every step or primarily just the first one?

Reply:
The problem is that in the micro configuration that shortened run of the appServerWarmer is leading to response-time problems in the initial portion of the first QoS period. For larger configurations we could probably have a shorter ramp-up.
We can experiment with these parameters when we do our sizing tests and see if there is a tradeoff between appServerWarmer time and rampUp time.  I agree that it would be better to have a shorter rampup, although given the start-up and shut-down time, a couple of minutes isn't a huge problem.

Reply:
Also, I'm not clear if this is what you were asking, but the rampUp is only before the first QoS period (at the start of the run).

Reply:
My point was that for a findMaxMultiRun, rampup happens for every step (sub run).